SPECS/fsverity-utils/fsverity-utils.spec

```diff
@@ -0,0 +1,116 @@
+Summary:        fsverity utilities
+Name:           fsverity-utils
+Version:        1.4
+Release:        8%{?dist}
+License:        BSD
```

Comment:
This is incorrect- the README states that this package is released under the MIT license, and the `LICENSE` file reflects that as well.

Reply:
Updated